azure-pipelines/test.runsettings

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<RunSettings>
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <DataCollector friendlyName="XPlat code coverage">
+        <Configuration>
+          <Format>cobertura</Format>
+          <Exclude>[coverlet.*.tests?]*,[*]Coverlet.Core*,[xunit.*]*</Exclude>
```

Comment:
@MarcoRossignoli this doesn't seem to do what I would expect. I still see lines in the cobertura.xml file that describe xunit assemblies, like this one:
```xml
    <package line-rate="0.24751439037153322" branch-rate="0" complexity="1204" name="xunit.runner.visualstudio.testadapter">
```

Reply:
After trying both the coverlet and built-in collector thoroughly, I succeeded first with the built-in one, so I guess I'm not using coverlet any more.